host/xtest/regression_4000_data.h

```diff
@@ -8650,4 +8650,94 @@ static const uint8_t x25519_shared_secret[] = {
 	0x76, 0xf0, 0x9b, 0x3c, 0x1e, 0x16, 0x17, 0x42
 };
 
+
```

Comment:
Please add a comment describing where you got these test vectors from.

Reply:
done in [21de224](https://github.com/OP-TEE/optee_test/pull/610/commits/21de2241a42fc9ab579909deb55e0653a6faf5bf)